src/main/java/com/clevertap/stormdb/index/IndexMap.java

```diff
@@ -0,0 +1,38 @@
+package com.clevertap.stormdb.index;
+
+public interface IndexMap {
+
+    /**
+     * Mapping having this as the key should not be allowed put since it represents null.
+     */
+    int NO_ENTRY_KEY = Integer.MAX_VALUE;
```

Comment:
You can use `StormDB#RESERVED_KEY_MARKER` instead of `Integer.MAX_VALUE`.

Reply:
Btw, that's not integer.maxvalue. Am I missing something?

Reply:
`Integer.MAX_VALUE` can be used as a legit key. Nothing should break.

Reply:
Good point. We should have done this much earlier :)